### DISCUSSION TITLE
Don't revalidate when focusing right after loading

### DISCUSSION THREAD

**Discussion by Linksku** [vercel/swr#1339] [Q&A]
I set `focusThrottleInterval` to 1 minute. If I fetch a resource, immediately unfocus, then refocus, SWR refetches. I think this is intended behavior, since the description for `focusThrottleInterval` is "only revalidate once during a time span". Technically, the initial fetch isn't a revalidation.

However, it makes sense to not refetch until 1 minute has passed since the initial fetch. Is this possible?

**Comment by shuding** [comment]
Can #1705 be a good solution for this?
